app/components/forms/wizard/other-details-step.js

```diff
@@ -164,6 +168,12 @@ export default Component.extend(FormMixin, EventWizardMixin, {
         this.sendAction('save');
       });
     },
+    unpublish() {
+      this.onValid(() => {
+        this.set('data.event.state', 'draft');
+        this.sendAction('save');
+     });
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 5.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6140894)